airbyte-integrations/connectors/source-metabase/source_metabase/streams.py

```diff
@@ -53,9 +62,100 @@ def path(self, **kwargs) -> str:
         return "dashboard"
 
 
-class Users(MetabaseStream):
+class Databases(MetabaseStream):
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        result = response.json().get("data", [])
+        yield from result
```

Comment:
```suggestion
   response_entity = "data"
```

The `MetabaseStream.parse_response`  is already parsing json and retrieving the `response_entity` field from it.